packages/suite-web/src/static/index.html

```diff
@@ -34,6 +34,21 @@
                 script.setAttribute('src','//cdn.crowdin.com/jipt/jipt.js');
                 document.head.prepend(script);
             }
+
```

Comment:
- We should not use const and arrow functions here.
- We should probably minimize it.

=> 

Do we need it here? What about moving it to codebase? I think user can see the wrong favicon for sec.

Reply:
@marekrjpolak implemented detection of OS theme

Reply:
not great, not terrible 🤔